src/status/libstatus/browser.nim

```diff
@@ -0,0 +1,22 @@
+import core, types, utils, strutils, strformat, json, chronicles
+
+proc storeBookmark*(url: string, name: string): Bookmark =
+  let payload = %* [{"url": url, "name": name}]
+  result = Bookmark(name: name, url: url)
+  try:
+    let resp = callPrivateRPC("browsers_storeBookmark", payload).parseJson["result"]
+    result.imageUrl = resp["imageUrl"].getStr
+  except:
+    discard
+
+proc updateBookmark*(ogUrl: string, url: string, name: string) =
+  let payload = %* [ogUrl, {"url": url, "name": name}]
+  discard callPrivateRPC("browsers_updateBookmark", payload)
```

Comment:
Can this rpc method fail somehow depending on the content of the payload?

Reply:
Hmm, good question. Maybe if it's a bad url
I'll add a try/catch